### PR TITLE
docs: suggest for analyze memory quota

### PR DESCRIPTION
**Pull request by chrysan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: statistics.md [hunk 1]
```diff
@@ -423,6 +423,24 @@ TiDB 支持持久化的配置项包括：
 - GlobalStats 会使用表级别的持久化配置
 - ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化
 
+
+### 统计信息收集的内存限制
+
+TiDB 从 v6.1.0 开始引入了统计信息收集的内存限制，当设置了 [`tidb_mem_quota_analyze`](/system-variables.md#tidb_mem_quota_analyze-从-v610-版本开始引入)，且统计信息收集任务（包括用户手动执行的 `ANALYZE` 和 TiDB 后台自动触发的）的总内存占用超过这个阈值时，用户执行的 `ANALYZE` 会被终止退出，并通过错误信息提示用户尝试更小的采样率或稍后重试。如果 TiDB 后台自动执行的统计信息更新任务因内存超限而退出，且使用的采样率高于默认值，则会使用默认采样率重试一次。当该变量值为负数或零时，TiDB 不对更新统计信息的前后台任务进行内存限制。
```
Comment (Oreoxmt) [inline statistics.md:429]:
```suggestion
TiDB 从 v6.1.0 开始引入了统计信息收集的内存限制，你可以通过 [`tidb_mem_quota_analyze`](/system-variables.md#tidb_mem_quota_analyze-从-v610-版本开始引入) 变量来控制 TiDB 更新统计信息时的最大总内存占用。
```

File: statistics.md [hunk 2]
```diff
@@ -423,6 +423,24 @@ TiDB 支持持久化的配置项包括：
 - GlobalStats 会使用表级别的持久化配置
 - ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化
 
+
```
Comment (Oreoxmt) [inline statistics.md:426]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]

**Review by xuyifangreeneyes — APPROVED** [review]

File: statistics.md [hunk 3]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
...
+> **注意：**
+>
+> 配置思路仅供参考，实际配置需要在真实场景中测试决定。
+
+- 最小值：需要大于集群上最宽的表采集统计信息时，TiDB 的最大内存使用。一个粗略的参考信息是，在测试集上，20 列的表在默认配置下，统计信息采集的最大内存使用约为 800 MiB；160 列的表在默认配置下，统计信息采集的最大内存使用约为 5 GiB。
+- 最大值：需要小于集群在不进行统计信息采集时的内存空余量。
+
+> **警告：**
+>
+> 目前 ANALYZE 的内存 quota 功能为实验特性，在生产环境中使用时可能存在内存统计有误差的情况。
```
Comment (TomShawn) [inline statistics.md:441]: 建议放到 第 427 行

File: statistics.md [hunk 4]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
...
 
+### 统计信息收集的内存限制
+
+TiDB 从 v6.1.0 开始引入了统计信息收集的内存限制，你可以通过 [`tidb_mem_quota_analyze`](/system-variables.md#tidb_mem_quota_analyze-从-v610-版本开始引入) 变量来控制 TiDB 更新统计信息时的最大总内存占用。
+
+`tidb_mem_quota_analyze` 的合理值配置与集群的数据相关，在使用默认采样率的情况下，主要和列的数量以及列上的值的大小相关，也和 TiDB 的内存配置相关，具体的配置思路如下：
+
+> **注意：**
+>
+> 配置思路仅供参考，实际配置需要在真实场景中测试决定。
```
Comment (TomShawn) [inline statistics.md:438]:
```suggestion
要合理地配置 `tidb_mem_quota_analyze` 值，你需要考虑集群的数据。在使用默认采样率的情况下，主要考虑列的数量、列上的值的大小，以及TiDB 的内存配置相关。你可参考以下方面来配置变量的最大值和最小值：

> **注意：**
>
> 以下配置建议仅供参考，实际配置需要在真实场景中测试确定。
```

File: statistics.md [hunk 5]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
...
+
+- 最小值：需要大于集群上最宽的表采集统计信息时，TiDB 的最大内存使用。一个粗略的参考信息是，在测试集上，20 列的表在默认配置下，统计信息采集的最大内存使用约为 800 MiB；160 列的表在默认配置下，统计信息采集的最大内存使用约为 5 GiB。
```
Comment (TomShawn) [inline statistics.md:440]: 看不懂「需要大于集群上最宽的表采集统计信息时，TiDB 的最大内存使用」是什么意思，表述不清

File: system-variables.md [hunk 6]
```diff
@@ -1313,6 +1301,22 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 该变量默认值为 `CANCEL`，但在 TiDB v4.0.2 及之前的版本中，默认值为 `LOG`。
 - 在 v6.1.0 之前这个开关通过 TiDB 配置文件 (`oom-action`) 进行配置，升级到 v6.1.0 时会自动继承原有设置。
 
+### `tidb_mem_quota_analyze` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
+> **警告：**
+>
+> 目前 ANALYZE 的内存 quota 功能为实验特性，在生产环境中使用时可能存在内存统计有误差的情况。
```
Comment (TomShawn) [inline system-variables.md:1308]:
```suggestion
> 目前限制 ANALYZE 的内存使用量为实验特性，在生产环境中使用时可能存在内存统计有误差的情况。
```

File: statistics.md (hunk 5 above)
Comment (Oreoxmt) [inline statistics.md:440]:
```suggestion
- 最小值：需要大于集群上列最多的表收集统计信息时，TiDB 的最大内存使用。一个粗略的参考信息是，在测试集上，20 列的表在默认配置下，统计信息收集的最大内存使用约为 800 MiB；160 列的表在默认配置下，统计信息收集的最大内存使用约为 5 GiB。
```

File: statistics.md [hunk 7]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
...
+- 最大值：需要小于集群在不进行统计信息采集时的内存空余量。
```
Comment (Oreoxmt) [inline statistics.md:441]:
```suggestion
- 最大值：需要小于集群在不进行统计信息收集时的内存空余量。
```

File: statistics.md (hunk 5 above)
Comment (Oreoxmt) [reply statistics.md]: > 看不懂「需要大于集群上最宽的表采集统计信息时，TiDB 的最大内存使用」是什么意思，表述不清

最小值需要大于集群上列最多的表收集统计信息时，TiDB 的最大内存使用

File: statistics.md [hunk 8]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
 - GlobalStats 会使用表级别的持久化配置
 - ANALYZE TABLE ... PARTITION ... WITH ... 所指定的分区配置会被忽略，且不会被持久化
 
+### 统计信息收集的内存限制
+
+> **警告：**
+>
+> 目前 ANALYZE 的内存 quota 功能为实验特性，在生产环境中使用时可能存在内存统计有误差的情况。
```
Comment (Oreoxmt) [inline statistics.md:430]:
```suggestion
> 目前限制 ANALYZE 的内存使用量为实验特性，在生产环境中使用时可能存在内存统计有误差的情况。
```

File: statistics.md [hunk 9]
```diff
@@ -423,6 +423,23 @@ TiDB 支持持久化的配置项包括：
...
+
+TiDB 从 v6.1.0 开始引入了统计信息收集的内存限制，你可以通过 [`tidb_mem_quota_analyze`](/system-variables.md#tidb_mem_quota_analyze-从-v610-版本开始引入) 变量来控制 TiDB 更新统计信息时的最大总内存占用。
+
+要合理地配置 `tidb_mem_quota_analyze` 值，你需要考虑集群的数据。在使用默认采样率的情况下，主要考虑列的数量、列上的值的大小，以及 TiDB 的内存配置。你可参考以下建议来配置变量的最大值和最小值：
+
+> **注意：**
+>
+> 以下配置建议仅供参考，实际配置需要在真实场景中测试确定。
+
+- 最小值：需要大于集群上列最多的表收集统计信息时，TiDB 的最大内存使用。一个粗略的参考信息是，在测试集上，20 列的表在默认配置下，统计信息收集的最大内存使用约为 800 MiB；160 列的表在默认配置下，统计信息收集的最大内存使用约为 5 GiB。
```
Comment (TomShawn) [inline statistics.md:440]:
```suggestion
- 最小值：需要大于 TiDB 从集群上列最多的表收集统计信息时使用的最大内存。一个粗略的参考信息是，在测试集上，20 列的表在默认配置下，统计信息收集的最大内存使用约为 800 MiB；160 列的表在默认配置下，统计信息收集的最大内存使用约为 5 GiB。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**